docs/site/index/downloads.md

```diff
@@ -13,74 +13,74 @@ The following table lists the files for each released version. For license infor
-| 1.6.1 | 2008-07-23 | v1.6.1 | [CLDR1.6.1](https://unicode.org/Public/cldr/1.6.1/) |   | [LDML1.6.1](https://www.unicode.org/reports/tr35/tr35-11.html) | [~~Δ1.6.1~~](https://unicode.org/cldr/trac/query?status=closed&col=id&col=summary&milestone=1.6.1) | release-1-6-1 |   | n/a |
-| 1.6 | 2008-07-02 | v1.6 | [CLDR1.6](https://unicode.org/Public/cldr/1.6.0/) |   | [LDML1.6](https://www.unicode.org/reports/tr35/tr35-10.html) | [~~Δ1.6~~](https://unicode.org/cldr/trac/query?status=closed&col=id&col=summary&milestone=1.6) | release-1-6 |   | n/a |
-| 1.5.1 | 2007-12-21 | v1.5.1 | [CLDR1.5.1](https://unicode.org/Public/cldr/1.5.1/) |   | [LDML1.5.1](https://www.unicode.org/reports/tr35/tr35-9.html) | [~~Δ1.5.1~~](https://unicode.org/cldr/trac/query?status=closed&col=id&col=summary&milestone=1.5.1) | release-1-5-1 |   | n/a |
-| 1.5 | 2007-07-31 | v1.5 | [CLDR1.5](https://unicode.org/Public/cldr/1.5.0/) |   | [LDML1.5](https://www.unicode.org/reports/tr35/tr35-8.html) | [~~Δ1.5~~](https://unicode.org/cldr/trac/query?status=closed&col=id&col=summary&milestone=1.5) | release-1-5 |   | n/a |
-| 1.4.1 | 2006-11-03 | v1.4.1 | [CLDR1.4.1](https://unicode.org/Public/cldr/1.4.1/) |   | [LDML1.4.1](https://www.unicode.org/reports/tr35/tr35-7.html) | [~~Δ1.4.1~~](https://unicode.org/cldr/trac/query?status=closed&col=id&col=summary&milestone=1.4.1) | release-1-4-1 |   | n/a |
-| 1.4 | 2006-07-17 | v1.4 | [CLDR1.4](https://unicode.org/Public/cldr/1.4.0/) |   | [LDML1.4](https://www.unicode.org/reports/tr35/tr35-6.html) | [~~Δ1.4~~](https://unicode.org/cldr/trac/query?status=closed&col=id&col=summary&milestone=1.4) | release-1-4 |   | n/a |
-| 1.3 | 2005-06-02 | v1.3 | [CLDR1.3](https://unicode.org/Public/cldr/1.3.0/) |   | [LDML1.3](https://www.unicode.org/reports/tr35/tr35-5.html) | [~~Δ1.3~~](https://unicode.org/cldr/trac/query?status=closed&col=id&col=summary&milestone=1.3) | release-1-3 |   | n/a |
-| 1.2 | 2004-11-04 | v1.2 | [CLDR1.2](https://unicode.org/Public/cldr/1.2.0/) |   | [LDML1.2](https://www.unicode.org/reports/tr35/tr35-4.html) | [~~Δ1.2~~](https://unicode.org/cldr/trac/query?status=closed&col=id&col=summary&milestone=1.2) | release-1-2 |   | n/a |
-| 1.1.1 | 2004-07-29 | n/a | [CLDR1.1.1](https://unicode.org/Public/cldr/1.1.1/) |   | [LDML1.1](https://www.unicode.org/reports/tr35/tr35-2.html) | n/a | release-1-1-1 |   | n/a |
-| 1.1 | 2004-06-08 | n/a | [CLDR1.1](https://unicode.org/Public/cldr/1.1.0/) |   | [LDML1.1](https://www.unicode.org/reports/tr35/tr35-2.html) | n/a | release-1-1 |  | n/a |
-| 1.0 | 2003-12-19 |   |    |see [CLDR 1.0 Release](https://unicode.org/cldr/repository_access.html#CLDR_1.0_Release)|  |  |release-1-0| | n/a |
+| Latest |  | [latest-version][] | [latest-data][] | [latest-charts][] | [latest-ldml][] | [latest-tickets][] | [release-46][]| [latest-delta-dtd][] | See release notes |
```

Comment:
I like the reorg.

I think the 'fake' links don't add any value, and just confuse people who click on them. What about making a subhead for each release in the sections below, and putting a link just on the item in the No. column.
```suggestion
| Latest[#latest] |  | latest-version | latest-data | latest-charts | latest-ldml | latest-tickets | release-46| latest-delta-dtd | See release notes |
```

Then down in your new sections, add ### headers that each line will link to.

Reply:
@macchiati not sure what you mean by "fake links" - they all work for me, see https://9aaa9b29.cldr.pages.dev/index/downloads for example

Reply:
You are not seeing what I see in the file diff, which is:
```
| Latest | | [latest-version][] | [latest-data][] | [latest-charts][] | [latest-ldml][] | [latest-tickets][] | [release-46][]| [latest-delta-dtd][] | See release notes |
```

Reply:
please note https://www.markdownguide.org/basic-syntax/#reference-style-links - these refer to URLs below.

Reply:
got it, thank! I'd never seen those before.